docs/tutorials/best-practices/speed-up-templates.md

```diff
@@ -0,0 +1,138 @@
+# Speed up your Coder templates and workspaces
+
+October 31, 2024
+
+---
+
+If it takes your workspace a long time to start, find out why and make some
+changes to your Coder templates to help speed things up.
+
+## Monitoring
+
+You can monitor [Coder logs](../../admin/monitoring/logs.md) through the
+system-native tools on your deployment platform, or stream logs to tools like
+Splunk, Datadog, Grafana Loki, and others.
+
+### Coder Observability Chart
+
+Use the [Observability Helm chart](https://github.com/coder/observability) for a
+pre-built set of monitoring integrations.
+
+With the Observability Helm chart, you can monitor [which specific startup
+metrics are a part of the Helm chart?]
+
+To install it with Helm:
+
+```shell
+helm repo add coder-observability https://helm.coder.com/observability
+helm upgrade --install coder-observability coder-observability/coder-observability --version 0.1.1 --namespace coder-observability --create-namespace
+```
+
+### Enable Prometheus metrics for Coder
+
+Our observability bundle gives you this for free which is nice and has
+instructions on how to do it with Coder
```

Comment:
Can we reword this? Also let's link to docs/admin/integrations/prometheus.md

Reply:
🫨 that's my fault for not cleaning up my notes correctly - this doesn't belong here